src/atrule/tailwind-import.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * @fileoverview Tailwind 4 `@import` rule parser
+ * @see https://github.com/eslint/csstree/blob/1d74355c1960315bf55a33b0652d13f97ebb1ba2/lib/syntax/atrule/import.js.
+ */
+
+//-----------------------------------------------------------------------------
+// Imports
+//-----------------------------------------------------------------------------
+
+import { tokenTypes } from "@eslint/css-tree";
+
+//-----------------------------------------------------------------------------
+// Type Definitions
+//-----------------------------------------------------------------------------
+
+/**
+ * @import { ParserContext as BaseParserContext, ConsumerFunction, SyntaxConfig } from "@eslint/css-tree";
+ */
+ 
+ /**
+  * @typedef {'Raw' | 'Layer' | 'Condition' | 'Declaration' | 'String' | 'Identifier' | 'Url' | 'Function' | 'MediaQueryList'} ConsumerNames
+  * @typedef {BaseParserContext & { [key in ConsumerNames]: ConsumerFunction }} ParserContext
+  */
+ 
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+/**
+ * @param {ConsumerFunction} parse
+ * @param {ConsumerFunction=} fallback
+ * @this {ParserContext}
+ */
+function parseWithFallback(parse, fallback) {
+    return this.parseWithFallback(
+        () => {
+            try {
+                return parse.call(this);
+            } finally {
+                this.skipSC();
+                if (this.lookupNonWSType(0) !== tokenTypes.RightParenthesis) {
+                    // @ts-expect-error
+                    this.error();
+                }
+            }
+        },
+        fallback || (() => this.Raw(null, true))
+    );
+}
+
+const parseFunctions = {
+    /**
+     * @this {ParserContext}
+     */
+    layer() {
+        this.skipSC();
+
+        const children = this.createList();
+        const node = parseWithFallback.call(this, this.Layer);
+
+        if (node.type !== 'Raw' || node.value !== '') {
+            children.push(node);
+        }
```

Comment:
I noticed this logic in a few places. CSSTree inserts `Raw` nodes where it can't figure out how to parse the source. I'm not sure we should exclude them here.